KinConnections/email_wrapper.py

```diff
@@ -0,0 +1,68 @@
+# email_wrapper.py - KinConnections
```

Comment:
Let's get rid of this file as well. Seems like you've moved all the functionality here.